iree/test/e2e/regression/generate_e2e_matmul_tests.py

```diff
@@ -263,14 +268,37 @@ class TestInputMatricesStaticShapes:
 # converts from the runtime shape dimensions in TestShape and given dynamicity to
 # the set of static shapes to be used in a test function's input tensors.
 def generate_static_shapes(shape: TestShape, dynamicity: Dynamicity):
```

Comment:
These aren't static shapes! :-P This keeps coming up. Maybe you mean to say that they're ranked? I think that's already suggested by shapes though, since unranked couldn't have a shape. I would just call it `generate shapes`

Reply:
Yeah like on the other PR, I was misusing the word 'static'.  Did a whole pass of fixing that on this whole file.